eth/roundinitializer.go

```diff
@@ -104,23 +103,7 @@ func (r *RoundInitializer) tryInitialize() error {
 	r.mu.Lock()
 	defer r.mu.Unlock()
 
-	currentL1Blk := r.tw.LastSeenL1Block()
-	lastInitializedL1BlkHash := r.tw.LastInitializedL1BlockHash()
-
-	epochSeed := r.currentEpochSeed(currentL1Blk, r.nextRoundStartL1Block, lastInitializedL1BlkHash)
-
-	ok, err := r.shouldInitialize(epochSeed)
```

Comment:
While the previous logic seemed too complex while at the same time didn't work in practice, I think not having any logic at all to prevent concurrent (or just repeated) round initialization is a big waste here.

While a single failed round initialization transaction is cheap ([$0.002](https://arbiscan.io/tx/0xe51fbb2b082020bb606b37ad849b1de82ed938b4174719a989d22232e5d2749c)), IMO it doesn't sound as cheap if we think about the global cost of having _several_ failed transactions for every round. With 100 orchestrators globally, plus 27 catalysts and 120 broadcasters, we'd basically be wasting $0.50 every day from our protocol, as well as polluting the onchain history for the round initializer contract.

I tried giving some thought on this and I think a relatively simple but potentially good enough solution could be:
- Add a randomized delay here, so we spread nodes across different times
  - This could be deterministic from the address instead, problem is all our Bs would end in the same bucket
  - I thought of a short delay from 0-10s or 0-20s (avg 10s). This should be enough for most nodes to realize the round is already initialized and avoid repeating it.
- Before actually initializing the round, check if it's already initialized first by calling `r.client.CurrentRoundInitialized()`. This is a read transaction that won't incur gas costs

WDYT?

Reply:
Yeah, good, let's introduce a random delay. I think it does not need to be deterministic, simple rand should work.